filepicker/src/main/java/dev/arkbuilders/components/filepicker/ArkFilePickerViewModel.kt

```diff
@@ -203,10 +203,14 @@ internal class ArkFilePickerViewModel(
         val haveRoot = haveRoot()
 
         root?.let {
-
             //Make sure file isn't inside a root folder
             if (root != file) {
-                val foundAsFavorite = favorites?.any { file.endsWith(it) } ?: false
+                val favorites = rootsWithFavorites.map { (root, relativeFavorites) ->
+                    relativeFavorites.map {
+                        root.resolve(it)
```

Comment:
But why `rootsWithFavorites.map` contains absolute paths? We should save only relative paths on disk.

Reply:
Favorites in `rootsWithFavorites` are relative and I map them into absolute paths for check

Reply:
@mdrlzy I'm curious why do we have to resolve relative path into absolute path just for checking if the current being pinned folder is already a favorite, while we still can do it with the original relative paths?

Reply:
I thought that we check all favorites and not just under specific root. I didn't like the check via `endsWith`. But in fact everything was ok, I returned it as it was.